src/write_riemann.c

```diff
@@ -625,7 +624,7 @@ static void wrr_free(void *p) /* {{{ */
 
   wrr_disconnect(host);
 
-  pthread_mutex_lock(&host->lock);
+  pthread_mutex_unlock(&cb->curl_lock);
   pthread_mutex_destroy(&host->lock);
```

Comment:
Changing lock to unlock looks like bug fix, but why the `host->lock` to  `cb->curl_lock` change, when only `host->lock` is destroyed (and other plugins destroy `cb` lock)?